packages/side-nav/src/SideNav.story.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { SideNav, SideNavItem, SideNavGroup } from './index';
+
+function MockAtlasSideNav() {
+  return (
+    <SideNav>
+      <SideNavGroup headerText="Data Storage">
+        <SideNavItem href="#clusters" active>
+          Clusters (active)
+        </SideNavItem>
+        {/* <SideNavItem>Triggers (not a link)</SideNavItem>
```

Comment:
These were commented out when testing something, and I accidentally committed the comments. These will be re-enabled.

Reply:
Can add knobs here, so that users can play with active/non-active, href/nohref and the text of the SideNavItem